src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,35 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hidden = false;
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(herbivore) {
+    if (herbivore.hidden === undefined || herbivore.hidden === true) {
+      return;
+    }
+
+    herbivore.health -= 50;
```

Comment:
```suggestion
    if (!herbivore.hidden) {
          herbivore.health -= 50;
    }
```

You can simplify this check. And also you need to check the instance of herbivore object, it should be the instance of Herbivore  class